docs/api/api.yml

```diff
@@ -0,0 +1,83 @@
+openapi: 3.0.0
+info:
+  title: Download Service API
+  version: 1.1.2
+  description: |
+    This document specifies the Download Service API. It provides a direct interface to download files from the archive.
+    This API implements the operations `GET` on the paths `/` and `/ark:/:NAAN/:uuid/*`.
+    The NAAN represents the configured 'name assigning authority number' which for DANS is 73189.
+    It is not neccessary for this service to supply the bag store the bag is stored in.
+
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0
+
+servers:
+  - url: 'http://localhost:20160'
+  - url: 'http://test.dans.knaw.nl:20160'
```

Comment:
this doesn't work anymore. The local VMs do not exist anymore

Reply:
Deze URL staat nog in meerdere projecten. Hebben we een alternatieve URL die we hier neer kunnen zetten? Ik had hem voor nu zo neergezet als zijnde 'de easy server waarmee je werkt'

Reply:
I'd suggest only keeping the `localhost` variant as '_the easy server you're working with_'

Reply:
You could use `http://localhost:20160` in a development-environment, if you configure the back-end servers to be on `deasy` and/or `ddark`. Other URLs that will be available are:

* https://datavault.dans.knaw.nl/
* https://demo.datavault.dans.knaw.nl/ (maybe).